src/components_redux/Input.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+
+class Input extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.openNav = this.openNav.bind(this);
+    //this.closeNav = this.closeNav.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener("click", this.closeNav);
```

Comment:
as discussed - why?
Also `this.closeNav` doesn't exists.

Reply:
I'll remove this.